pkgs/applications/office/gnote/default.nix

```
@@ -0,0 +1,43 @@
+#{ stdenv, lib, meson }:
+with import <nixpkgs> {};
+stdenv.mkDerivation rec {
+  pname = "gnote";
+  version = "42.0";
+
+  src = pkgs.fetchurl {
+    url = "https://gitlab.gnome.org/GNOME/gnote/-/archive/${version}/gnote-${version}.tar.gz";
```

You can use a GNOME mirror like this:

```suggestion
    url = "mirror://gnome/sources/gnote/${lib.versions.major version}/gnote-${version}.tar.xz";
```